style.css

```diff
@@ -132,138 +121,84 @@ body {
   display: grid;
   grid-template-columns: 1fr;
   align-items: center;
-  gap: 5px;
-  width: 375px;
-  left: 0;
-  margin-top: 682px;
 }
 
 .recent-work {
   display: grid;
   gap: 24px;
   justify-items: center;
   text-align: justify;
+  padding: 0 24px 0 24px;
+  margin: 120px 24px 0 24px;
+}
+
+.most-recent {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-bottom: 80px;
+  gap: 24px;
 }
```

Comment:
- [ ] As requested by the previous CR kindly adjust the title to match the Figma template 👇🏽 

>  👍 Well done on the titles! However, to better align with the design, please add proper font styles and colors. Consider adjusting the font size, font weight, and other relevant properties to match the design specifications. By implementing the appropriate font styles, your titles will look more polished and visually consistent with the overall design. Keep up the good work, and I'm excited to see your project take on a more refined appearance! 🎨📝🚀

| Figma design | Your design |
|---|---|
| ![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/58642949/08a47014-406f-47dc-87ec-5d97a36f3008) | ![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/58642949/d4ffe750-7c1e-45e6-8b80-578620965dbf) |